### PR TITLE
Added the new changes for 1.9 Pipelines RN

### PR DESCRIPTION
**Pull request by Dhruv-Soni11**
Purpose: To resolve the following issue:

https://issues.redhat.com/browse/RHDEVDOCS-4891

Aligned team: DevTools

OCP version this PR applies to (cherrypicking): enterprise 4.11 and later

Content for preview: https://55040--docspreview.netlify.app/openshift-enterprise/latest/cicd/pipelines/op-release-notes.html#new-features-1-9_op-release-notes

SME review : @vdemeester @savitaashture @concaf

QE review: @ppitonak @VeereshAradhya

PM review: @siamaksade 

Peer review: @rolfedh

### CODE REVIEW

**Comment by Dhruv-Soni11** [comment]
Hi @vdemeester @rupalibehera @siamaksade - I have made the changes as per your suggestion in the content. Have a look and provide your feedback, if any. Feel free to tag anyone so that they can oversee the changes too.

cc @Preeticp @pmkovar @shipsing: Tagging you folks here for reference.

**Review by vdemeester — APPROVED** [review]

**Review by ppitonak — APPROVED** [review]
LGTM

**Comment by koustavsaha** [comment]
LGTM

File: modules/op-release-notes-1-9.adoc
Comment (savitaashture) [inline modules/op-release-notes-1-9.adoc:80]: I am thinking we should avoid mentioning about `TektonAddon`  as it may confuse to new PAC user 

@vdemeester wdyt for below sentence 
```suggestion
* With this update, {pac} is installed by default. You can disable {pac} by using the following flag:
```

Comment (Preeticp) [reply modules/op-release-notes-1-9.adoc:80]: @dhruv, please see the guidance wrt Kubernetes Objects here: https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#object-references especially the second note and the examples above it. That is relevant if you retain the words - `TektonAddon` If not please read and apply when you use CRDs.

Comment (dhruv) [reply modules/op-release-notes-1-9.adoc:80]: Wrong dev

Comment (Dhruv-Soni11) [reply modules/op-release-notes-1-9.adoc:80]: @Preeticp - I have decided to remove the words TektonAddon as instructed by the SME for the time being.

**Comment by savitaashture** [comment]
LGTM

File: modules/op-release-notes-1-9.adoc
Comment (jkandasa) [inline modules/op-release-notes-1-9.adoc:86]: @Dhruv-Soni11 In OpenShift doc, we have to use `oc` command

Comment (savitaashture) [reply modules/op-release-notes-1-9.adoc:86]: I see everywhere its used `oc` in Openshift doc but ya will defer to @sounix000 input

Comment (Preeticp) [reply modules/op-release-notes-1-9.adoc:86]: +1 to @jkandasa's comment. Please change any instance of `kubectl` to `oc`

Comment (Dhruv-Soni11) [reply modules/op-release-notes-1-9.adoc:86]: @Preeticp - I see from @sounix000's status is OOO till 29th Jan. Can we ask some other writer or should I go ahead and change any instances of `kubectl` to `oc` in this command?

Comment (Dhruv-Soni11) [reply modules/op-release-notes-1-9.adoc]: Thank you for informing me about this change, Preeti. I have gone ahead and replaced an instance of `kubectl` to `oc` in the content. @VeereshAradhya @ppitonak and @rupalibehera, could you confirm if this command is accurate after this change.

File: modules/op-release-notes-1-9.adoc
Comment (chmouel) [reply modules/op-release-notes-1-9.adoc:289]: should be fine! 👍🏻

Comment (Dhruv-Soni11) [reply modules/op-release-notes-1-9.adoc:289]: @VeereshAradhya @rupalibehera @ppitonak - Just updating this thread with a gentle reminder for your inputs. PTAL.

**Comment by sounix000** [comment]
@Dhruv-Soni11 - Generally, `kubectl` and `oc` are 100% compatible. It is usually safe to substitute `kubectl` with `oc` in the documentation.

**Comment by vdemeester** [comment]
Still LGTM

**Review by ppitonak — COMMENTED** [review]
LGTM

**Comment by Dhruv-Soni11** [comment]
@pmkovar - Could you look at the changes and approve it?

File: modules/op-release-notes-1-9.adoc
Comment (Dhruv-Soni11) [inline modules/op-release-notes-1-9.adoc:258]: @rupalibehera  @ppitonak - I have added the inputs as per your feedback. Kindly have a look and let me know your comments, if any.

File: modules/op-release-notes-1-9.adoc
Comment (rupalibehera) [reply modules/op-release-notes-1-9.adoc:292]: looks good

File: modules/op-release-notes-1-9.adoc
Comment (rupalibehera) [reply modules/op-release-notes-1-9.adoc:258]: @vdemeester could you please check if this looks correct

Comment (vdemeester) [reply modules/op-release-notes-1-9.adoc:258]: I think that is correct, not sure exactly how is the wording around this in the rest of the openshift docs, but it seems alright there.

**Comment by pmkovar** [comment]
LGTM, thanks

File: modules/op-release-notes-1-9.adoc
Comment (rolfedh) [inline modules/op-release-notes-1-9.adoc:256]: Maybe, for minimalism:
```suggestion
* With this update, the `tkn-pac setup` command has been removed. Instead, use the `tkn-pac webhook add` command to re-add a webhook to an existing Git repository. And use the `tkn-pac webhook update-token` command to update the personal provider access token for an existing Secret object in the Git repository.
```

File: modules/op-release-notes-1-9.adoc
Comment (rolfedh) [inline modules/op-release-notes-1-9.adoc:311]: If I were a user, I might have some questions when I read this RN:
- Where or how do I find these fields you're talking about?
- Which fields were previously unsettable and are now settable?
- Are **some** fields still unsettable?  
- Where can I see the default values?

Maybe something more like this:
```suggestion
* Before this update, you could not set some `timeouts` fields in the <name of resource or config file>. This update resolves the issue. Now, you can set the values of the following `timeouts` fields: <field a>, <field b>, and <field c>. If you don't set a value for these fields, they use default values. To learn more, see: link:<URL etc.>
```

**Review by rolfedh — COMMENTED** [review]
Nice work, Dhruv. I've offered some questions and suggestions. HTH!

File: modules/op-release-notes-1-9.adoc
Comment (rolfedh) [reply modules/op-release-notes-1-9.adoc:86]: I believe the extra blank lines break the alignment of the command with the bullet point. To catch formatting/layout issues, I recommend examining the preview of your work. We have some PR review checklists that help us walk through all the correct steps. @bburt-rh has a nice easy one. Brian, can  you share the link here?

Comment (bburt-rh) [reply modules/op-release-notes-1-9.adoc:86]: @rolfedh @Dhruv-Soni11 - I added the link to my checklist in the Slack review request for this PR

Comment (bburt-rh) [inline modules/op-release-notes-1-9.adoc:86]:
```suggestion
$ oc patch tektonconfig config --type="merge" -p '{"spec": {"platforms": {"openshift":{"pipelinesAsCode": {"enable": false}}}}}'
```

- Also the CLI command needs to have a prompt that indicates the proper level of permissions to run it.

File: modules/op-release-notes-1-9.adoc
Comment (Dhruv-Soni11) [reply modules/op-release-notes-1-9.adoc:311]: @vdemeester @rupalibehera - Could you guys help me with some inputs here please?

File: modules/op-release-notes-1-9.adoc
Comment (Dhruv-Soni11) [reply modules/op-release-notes-1-9.adoc:256]: Fixed.

File: modules/op-release-notes-1-9.adoc
Comment (Dhruv-Soni11) [reply modules/op-release-notes-1-9.adoc:80]: Fixed.

File: modules/op-release-notes-1-9.adoc
Comment (Dhruv-Soni11) [reply modules/op-release-notes-1-9.adoc:86]: Fixed.

File: modules/op-release-notes-1-9.adoc
Comment (rolfedh) [reply modules/op-release-notes-1-9.adoc:80]: Another possibility, if you encounter a similar construction in the future: 
```
To disable {pac}, enter:
```

File: modules/op-release-notes-1-9.adoc
Comment (Dhruv-Soni11) [reply modules/op-release-notes-1-9.adoc]: Fixed this comment in the latest iteration. Kindly check and confirm if it is fine.

File: modules/op-release-notes-1-9.adoc
Comment (vdemeester) [reply modules/op-release-notes-1-9.adoc:309]: The fix is https://github.com/tektoncd/pipeline/pull/5460 .
> This commit updates PipelineRun validation to allow a user to specify only a tasks timeout or only a finally timeout.
Prior to this commit, if a user wanted to specify no Pipeline timeout (i.e. timeouts.pipeline = 0), they would not be able to specify values for the other timeouts fields.

This is on PipelineRun indeed 👼🏼

Comment (Dhruv-Soni11) [reply modules/op-release-notes-1-9.adoc:309]: Thanks for providing feedback folks. I have updated the content. PTAL. Thanks.

File: modules/op-release-notes-1-9.adoc
Comment (Dhruv-Soni11) [reply modules/op-release-notes-1-9.adoc:312]: Yes, this would be duplicate content. I am going to delete this sentence.

File: modules/op-release-notes-1-9.adoc
Comment (Dhruv-Soni11) [reply modules/op-release-notes-1-9.adoc]: Fixed as per your suggestion @rolfedh. PTAL.

File: modules/op-release-notes-1-9.adoc
Comment (rolfedh) [reply modules/op-release-notes-1-9.adoc]: @dhruv Based on your examination of the preview, you can mark this as resolved if the formatting looks good now.

File: modules/op-release-notes-1-9.adoc
Comment (rolfedh) [reply modules/op-release-notes-1-9.adoc]: Outdated. See following comment.

File: modules/op-release-notes-1-9.adoc
Comment (rolfedh) [reply modules/op-release-notes-1-9.adoc]: LGTM - needs SME review @vdemeester

File: modules/op-release-notes-1-9.adoc
Comment (rolfedh) [reply modules/op-release-notes-1-9.adoc]: LGTM - needs SME review @vdemeester

**Review by rolfedh — APPROVED** [review]
LGTM. 
Still needs @vdemeester 's review of the updated release note before merge.

File: modules/op-release-notes-1-9.adoc
Comment (Dhruv-Soni11) [reply modules/op-release-notes-1-9.adoc]: Hi @rolfedh - The content looks fine to me in the preview. Adding a screenshot for our reference. 
Screenshot from 2023-02-08 17-03-34

**Comment by Dhruv-Soni11** [comment]
> LGTM. Still needs @vdemeester 's review of the updated release note before merge.

**Review by vdemeester — APPROVED** [review]
LGTM

File: modules/op-release-notes-1-9.adoc
Comment (vdemeester) [reply modules/op-release-notes-1-9.adoc]: Yes 👍🏼

**Pull request merged**
